<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stuInfoForm</class>
 <widget class="QWidget" name="stuInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>280</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>管理</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="4">
       <widget class="QComboBox" name="comboBox_ItemStudent">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="placeholderText">
         <string>学生</string>
        </property>
        <item>
         <property name="text">
          <string>我的学生</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>所有学生</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QLineEdit" name="lineEdit_Select"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QComboBox" name="comboBox_ItemSelect">
        <item>
         <property name="text">
          <string>学号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>姓名</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>班级</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QQ号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>用户名</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButton_Select">
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>26</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0" colspan="4">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_stuNum">
          <property name="text">
           <string>学号:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_stuNum"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_stuName">
          <property name="text">
           <string>姓名：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_stuName"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_stuSex">
          <property name="text">
           <string>性别：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_stuSex"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_stuClass">
          <property name="text">
           <string>班级：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_stuClass"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_stuQQ">
          <property name="text">
           <string>QQ：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_stuQQ"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_stuUserName">
          <property name="text">
           <string>用户名：</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_stuUserName"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_stuPassWord">
          <property name="text">
           <string>密码：</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEdit_stuPassWord"/>
        </item>
       </layout>
      </item>
      <item row="5" column="1" colspan="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QPushButton" name="pushButton_modify">
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="pushButton_sort">
        <property name="text">
         <string>排序</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2" colspan="2">
       <widget class="QCheckBox" name="checkBox_sortOder">
        <property name="text">
         <string>升序</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="2">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>88</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QTableView" name="tableView">
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_sum">
         <property name="text">
          <string>总数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_sumNum">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_count">
         <property name="text">
          <string>计数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_countNum">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_row">
         <property name="text">
          <string>行：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_rowNum">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_col">
         <property name="text">
          <string>列：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_colNum">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>428</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
